lib/cutting_edge/workers/badge.rb

```diff
@@ -1,20 +1,44 @@
-require File.expand_path('../../badge.rb', __FILE__)
 require File.expand_path('../helpers.rb', __FILE__)
 
+module CuttingEdge
+  BADGE_TEMPLATE = File.read(File.expand_path('../../templates/badge.svg.erb', __FILE__)) unless defined?(BADGE_TEMPLATE)
+  BADGE_OK = File.read(File.expand_path('../../static/ok.svg', __FILE__)) unless defined?(BADGE_OK)
+  BADGE_ERROR = File.read(File.expand_path('../../static/error.svg', __FILE__)) unless defined?(BADGE_ERROR)
+  BADGE_BASE_WIDTH = 25 unless defined?(BADGE_BASE_WIDTH)
+  BADGE_CELL_WIDTH = 25 unless defined?(BADGE_CELL_WIDTH)
+  BADGE_LAYOUT = {
```

Comment:
Is this `LAYOUT` or would `BADGE_COLORS` be better?

Reply:
The colors themselves are great, btw. 👍

Reply:
I wanted to use it both to define the colors and the order of the colors at the same time. But now I wonder: since Hash order is undefined, I guess `BADGE_LAYOUT.keys` technically won't have a guaranteed order either?

Reply:
Just added a separate `BADGE_LAYOUT` Array 👍